store/store_download.go

```diff
@@ -50,12 +50,11 @@ import (
 	"github.com/snapcore/snapd/snapdtool"
 )
 
-var downloadRetryStrategy = retry.LimitCount(7, retry.LimitTime(90*time.Second,
-	retry.Exponential{
+var downloadRetryStrategy = retry.LimitCount(7, retry.Exponential{
```

Comment:
this needs to be formatted to be compatible with go 1.9, it seems it is not:

```diff
/home/runner/work/snapd/snapd/src/github.com/snapcore/snapd/store/store_download.go
--- /home/runner/work/snapd/snapd/src/github.com/snapcore/snapd/store/store_download.go.orig	2021-04-30 02:01:17.703122506 +0000
+++ /home/runner/work/snapd/snapd/src/github.com/snapcore/snapd/store/store_download.go	2021-04-30 02:01:17.703122506 +0000
@@ -51,9 +51,9 @@
 )
 
 var downloadRetryStrategy = retry.LimitCount(7, retry.Exponential{
-		Initial: 500 * time.Millisecond,
-		Factor:  2.5,
-	},
+	Initial: 500 * time.Millisecond,
+	Factor:  2.5,
+},
 )
 
 var downloadSpeedMeasureWindow = 5 * time.Minute

```

Reply:
I fixed the indenting in 744d080.